<launch>

  <arg name="debug" default="false"/>
  <arg if="$(arg debug)" name="launch_prefix" value="xterm -rv -e gdb -ex run -args"/>
  <arg unless="$(arg debug)" name="launch_prefix" value=""/>

  
 
  <node pkg="task_recorder" name="TaskRecorder" type="task_recorder_node" output="screen" launch-prefix="$(arg launch_prefix)">
    <!-- <env name="ROSCONSOLE_CONFIG_FILE"
       value="$(find task_recorder)/custom_rosconsole.config"/> -->
    <rosparam command="load" file="$(find task_recorder)/config/task_recorder_manager.yaml" />
    <rosparam command="load" file="$(find task_recorder)/config/task_recorder_db.yaml" />
    <!-- <rosparam command="load" file="$(find task_recorder)/config/task_recorder.yaml" /> -->
    <rosparam command="load" file="$(find task_recorder)/config/joint_states_recorder.yaml" />
  </node>
</launch>
